src/legacy/ui/public/private/private.js

```diff
@@ -108,98 +108,100 @@ function name(fn) {
   return fn.name || fn.toString().split('\n').shift();
 }
 
-uiModules.get('kibana/private')
-  .provider('Private', function () {
-    const provider = this;
-
-    // one cache/swaps per Provider
-    const cache = {};
-    const swaps = {};
+export function PrivateProvider() {
```

Comment:
Same as for the top nav directive. Private isn't used in this PR but it will become necessary for shimming discover and others in the same style